addons/skin.estuary/xml/DialogSelect.xml

```diff
@@ -4,11 +4,12 @@
 	<include>Animation_DialogPopupOpenClose</include>
 	<depth>DepthOSD</depth>
 	<controls>
-		<include condition="![Window.IsActive(gamesaves) | Window.IsActive(gamestretchmode) | Window.IsActive(gamevideofilter) | Window.IsActive(gamevideorotation) | Window.IsActive(ingamesaves)]">DefaultDialogSelectLayout</include>
+		<include condition="![Window.IsActive(selectvideoversion) | Window.IsActive(selectvideoextra) | Window.IsActive(gamesaves) | Window.IsActive(gamestretchmode) | Window.IsActive(gamevideofilter) | Window.IsActive(gamevideorotation) | Window.IsActive(ingamesaves)]">DefaultDialogSelectLayout</include>
 		<include condition="Window.IsActive(gamesaves)">GameDialogSelectSaveLayout</include>
 		<include condition="Window.IsActive(gamevideofilter)">GameDialogSelectFilterLayout</include>
 		<include condition="Window.IsActive(gamestretchmode)">GameDialogSelectViewLayout</include>
 		<include condition="Window.IsActive(gamevideorotation)">GameDialogSelectViewLayout</include>
 		<include condition="Window.IsActive(ingamesaves)">GameDialogSelectSaveInGameLayout</include>
+		<include condition="Window.IsActive(selectvideoversion)">VideoSelectLayout</include>
```

Comment:
Has `selectvideoextra` window ID been removed entirely? I haven't had a chance to test the very most recent changes yet but I thought it still existed from info dialog -- or does extras only use manage dialog now?

If it is still a valid window ID then need to fix up one of the conditions here -- currently the conditions mean no layout at all for `selectvideoextra`

Reply:
> Has `selectvideoextra` window ID been removed entirely?

No, it's still there.

Reply:
It's still there https://github.com/xbmc/xbmc/blob/7413267da70dbb26dcdfbe5e86244343c3716de7/xbmc/guilib/WindowIDs.dox#L137 but it's no longer used. For v21 the only way to access Extras is now via the Manage dialog (DialogVideoManager). Extras will be revisited in v22 so not sure if that's why it remains as a Window ID. Perhaps @ksooo can say if it's being kept to save having to readd in v22 for whatever is planned.